modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java

```diff
@@ -134,7 +134,8 @@ public KafkaToIgniteCdcStreamerApplier(
         int kafkaPartTo,
         Set<Integer> caches,
         int maxBatchSize,
-        AtomicBoolean stopped
+        AtomicBoolean stopped,
+        int kafkaReqTimeout
```

Comment:
Looks like javadoc for the new parameter not added.

Reply:
Done.